src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandler.java

```diff
@@ -54,7 +54,7 @@ public String camundaActivityId(CallbackParams callbackParams) {
     private CallbackResponse notifyApplicantForHwFOutcome(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
         CaseEvent hwfEvent = caseData.getHwFEvent();
-        if (Objects.nonNull(caseData.getApplicant1Email())) {
+        if (Objects.nonNull(caseData.getApplicant1Email()) && CaseEvent.FULL_REMISSION_HWF != hwfEvent) {
```

Comment:
why this change is here, we wouldn't know background why we did this here

Reply:
as discussed added a 1line comment explaining the reason.